packages/mobile/src/navigator/TabNavigator.tsx

```diff
@@ -95,22 +97,23 @@ export const TabNavigator = createBottomTabNavigator(
       },
     },
     [Screens.Send]: {
-      screen: Send,
+      screen: SendStack,
       navigationOptions: {
         tabBarButtonComponent: TabBarButtonComponent,
         tabBarIcon: (props: any) => <PaymentsIcon color={props.tintColor} />,
-        tabBarLabel: ({ tintColor }: LabelProps) => {
-          return <MenuText testID="SendNavigator" transKey="payments" tintColor={tintColor} />
+        tabBarLabel: () => null,
+        tabBarOnPress: () => {
```

Comment:
Why is this necessary? Doesn't it do this by default based on the `screen` property above

Reply:
We wanted to switch back to the "modal" view, so it should pop on the stack again and not actually switch tabs